src/Microsoft.Azure.ServiceBus/Primitives/ServiceBusConnection.cs

```diff
@@ -100,5 +100,9 @@ static void CloseConnection(AmqpConnection connection)
 
             return connection;
         }
+
+        /// <summary>Namespace name used for <see cref="ServiceBusConnection"/>.</summary>
+        /// <value>Namespace name.</value>
+        internal string NamespaceName => this.Endpoint.Host.Split(new[] {'.'}, StringSplitOptions.RemoveEmptyEntries)[0];
```

Comment:
Based on #196, you'll have to change this to show the entire endpoint and not just the namespace... :)